system/COPY/etc/motd

```diff
@@ -0,0 +1,13 @@
+Welcome to the
+                   _ _
+  __ _ _ __  _ __ | (_) __ _ _ __   ___ ___
+ / _` | '_ \| '_ \| | |/ _` | '_ \ / __/ _ \
+| (_| | |_) | |_) | | | (_| | | | | (_|  __/
+ \__,_| .__/| .__/|_|_|\__,_|_| |_|\___\___|
+      |_|   |_|            _
+  ___ ___  _ __  ___  ___ | | ___
+ / __/ _ \| '_ \/ __|/ _ \| |/ _ \
+| (_| (_) | | | \__ \ (_) | |  __/
+ \___\___/|_| |_|___/\___/|_|\___|
```

Comment:
I just want to address Jason's concern that users don't know what to do after logging into / ssh-ing into an appliance. [#792](https://github.com/ManageIQ/manageiq/pull/792#discussion_r18686268)

If you think using another font or different text will help you can test other fonts [here](http://patorjk.com/software/taag/#f=Ogre&t=appliance%0Aconsole). If you think there are better approaches, please let me know.